docs/api.md

```diff
@@ -0,0 +1,41 @@
+<!-- TOC -->
+
+- [1. API Documentation](#1-api-documentation)
+  - [1.1. POST /api/tests](#14-post-apitests)
+
+## 1.1. POST /api/tests
+
+Creates a synthetic test
+
+### 1.1.1. Expected Payload
+
+```json
+{
+  "test": {
+    "title": "My new test",
+    "locations": ["us-east-1"],
+    "minutesBetweenRuns": 60,
+    "type": "API",
+    "http_request": {
+      "http_method": "GET",
+      "url": "https://mysite.com/api/users",
+      "headers": {},
+      "body": {},
+      "assertions": {
+        "status_code": 200,
+        "contains_properties": []
```

Comment:
Addressed in the PR summary. Tim suggested a better design in the architecture docs that allows for different types of comparisons (e.g. equal_to, greater_than, etc.). I left it this way for this PR since this is what the current `testRunnerV0` expects